### PR TITLE
Bluetooth samples: add board conf for ISO_Broadcast

### PR DESCRIPTION
**Pull request by kaiser-ren**
With the specific configuration, the boards are able to broadcast Broadcast Isochronous Stream.

Signed-off-by: Kai Ren <renkaikaiser@163.com>

### CODE REVIEW

**Review by Thalley — DISMISSED** [review]

File: samples/bluetooth/iso_broadcast/boards/nrf52840dk_nrf52840.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_ADV_ISO=y
```
Comment (asbjornsabo) [inline samples/bluetooth/iso_broadcast/boards/nrf52840dk_nrf52840.conf:3]: Is the
`CONFIG_BT_CTLR_SYNC_PERIODIC` also needed?

See tests/bluetooth/shell/boards/*conf

**Comment by Thalley** [comment]
I guess this is related to https://github.com/zephyrproject-rtos/zephyr/issues/41635 which is perhaps better solved with https://github.com/zephyrproject-rtos/zephyr/issues/41635

**Comment by kaiser-ren** [comment]
> I guess this is related to https://github.com/zephyrproject-rtos/zephyr/issues/41635 which is perhaps better solved with https://github.com/zephyrproject-rtos/zephyr/pull/41876

I saw #41876 updated the ReadMe, good job. What are the next steps for this issue and related PR? Close them?

**Comment by Thalley** [comment]
> > I guess this is related to #41635 which is perhaps better solved with #41876
> 
> I saw #41876 updated the ReadMe, good job. What are the next steps for this issue and related PR? Close them?

The issue will be automatically closed when my PR is merged. 
This PR should be closed.

**Pull request closed without merging**